htdocs/js/jsx/behavioural_feedback_panel.jsx

```diff
@@ -0,0 +1,606 @@
+var SliderPanel = React.createClass({
+  render: function() {
+      return <div className="panel-group" id="bvl_feedback_menu">
+      <div className="breadcrumb-panel"><a className="info">Feedback for PSCID: {this.props.pscid}</a></div>
+    {this.props.children}
+    </div>
+  }
+});
+
+var FeedbackPanelContent = React.createClass({
+  propTypes: {
+    feedback_level : React.PropTypes.string.isRequired,
+    feedback_values : React.PropTypes.array
+  },
+  getInitialState: function(){
+    return {
+      currentEntryToggled: null
```

Comment:
Based on the name, I'd expect the default to be either true or false, not null..